tests/testIntegration.nim

```diff
@@ -89,20 +88,23 @@ ethersuite "Integration tests":
       client.post(baseurl1 & "/upload", "some file contents").body
 
     proc buy(cid: string): string =
-      let expiry = ((waitFor provider.currentTime()) + 10).toHex
+      when defined(windows):
```

Comment:
On both my local mac laptop and mac mini, with `let expiry = ((waitFor provider.currentTime()) + 30).toHex` that test takes ~30 seconds to complete, while with `10` instead of `30` it takes ~10 seconds to complete. I tried changing it back-and-forth a few times and the behavior is consistent.

I observe roughly the same behavior on my local Windows laptop/vm. However, Windows in (GitHub Actions) seems to be particularly slow (maybe because of vm/container contention), which is why I widened it for Windows.

I haven't dug into it more deeply and haven't compared with behavior on Linux, but it does seem a greater `expiry` value causes the test to take longer on some platforms.

Reply:
You're right! I'm sorry, I was thinking of the new behavior that I'm working on, but that's not in main yet. Sorry for the noise.